### DISCUSSION TITLE
Rules From Git Repo

### DISCUSSION THREAD

**Discussion by louzadod** [jertel/elastalert2#528] [Q&A] [answered]
Hi,
is there anyway, using the current ElastAlert helm chart, to add another container in the same ElastAlert pod (sidecar) in order to provide the rules? 

My rules are stored in git.
Thanks.

**Comment by jertel** [comment, marked as answer]
Not without modifications to the Helm chart. This discussion provided a high-level walk through on how to do this: https://github.com/jertel/elastalert2/discussions/507#discussioncomment-1494866

If you submit a PR for the `extraContainers` support we can get it into the official chart.
